### ISSUE TITLE
Shaping Our Work Culture

### ISSUE THREAD

**Issue by OleMussmann** [nlesc-sigs/soft-skills-sig#35]
What makes you productive? Gets you in the flow? Makes you happy? Work culture is something we primarily absorb from our surroundings, shaping our ideas of what an ideal workplace should look like. But what are our _actual_ needs? Are you in touch with them? What might the needs of your colleagues be? Of the eScience Center?

Let's shelve our preconceptions and imagine together all the large and small things that can shape our work experience. Realistic, unrealistic, fantastic; bring it on. We can never make everybody happy, but maybe we nudge our work culture in a direction it benefits most.

**Comment by PabRod** [comment]
Regarding this topic, I found the concepts in _Drive_, a book by Daniel H. Pink, pretty illuminating. He suggests that all motivation, and thus productivity of the best kind, rests in 3 pillars: autonomy, mastery, and purpose.

Just leaving it here as a possible conversation starter...

**Comment by lopeztarifa** [comment]
yep, Yifat talked about the book years ago, there is a recording somewhere. I read it too, and I like it. As a hypothesis on paper resonates with me. My problem is how to make it work with the constraints of reality.

**Comment by PabRod** [comment]
> My problem is how to make it work with the constraints of reality.

That's a good definition of both management and engineering :smile:

**Comment by lopeztarifa** [comment]
Also with the dynamics of life and human nature, the definition of purpose outlined in the book changes, at least to me. Anyway, it's a good start assuming there is a "Drive" that makes us productive, gets us in the flow, makes us happy (... as Ole says).

**Comment by OleMussmann** [comment]
Session done 🙂
